core-helm-chart/templates/operator.yaml

```diff
@@ -202,6 +202,7 @@ spec:
         - --leader-elect
         - --wds-name={{.Values.ControlPlaneName}}
         - --api-groups={{.Values.APIGroups}}
+        - -v=2
```

Comment:
Is the intent to allow changing verbosity only editing the deployment at runtime ? Or should we expose this at the helm chart values level ?

Reply:
This is a matter for future PRs. I definitely want to give deployers more control over verbosity. I also suspect we will want to be a little more organized and principled about how we use various verbosity levels.